src/helpers/expense.helpers.js

```diff
@@ -0,0 +1,56 @@
+const userService = require('../services/user.service');
+const {
+  USER_NOT_FOUND_ERROR,
+  AMOUNT_REQUIRED_ERROR,
+  USER_ID_REQUIRED_ERROR,
+  TITLE_REQUIRED_ERROR,
+  CATEGORY_REQUIRED_ERROR,
+} = require('../utils/config');
+
+const isUserExist = (userId, res) => {
+  const user = userService.getUserById(userId);
+
+  if (!user) {
+    res.status(400).send(USER_NOT_FOUND_ERROR);
```

Comment:
- helpers shouldn't know anything about statuses and what response to send, just validate data and throw an error if needed.
- function name should start from verb